### DISCUSSION TITLE
⭐⭐ Day 13 ⭐⭐

### DISCUSSION THREAD

**Discussion by CarolineMorton** [CarolineMorton/AdventOfCode_WIR#16] [Ideas]
⭐⭐ **Day 13** ⭐⭐

How is everyone doing today?

**Comment by AR2202** [comment]
[Day 13](https://github.com/AR2202/AdventOfCode_WIR/blob/main/day_13/src/main.rs)

**Comment by Sowmiya07** [comment]
[Here](https://github.com/Sowmiya07/AdventOfCode_WIR/blob/main/day_13/src/main.rs) is my Day 13 Part 1 solution.
